TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,71 @@
+desc "Bump the iOS and Android version with a commit and tag."
+lane :bump_version do |options|
+  type = options.fetch(:type)
+
+  case type
+  when "major", "minor", "patch"
```

Comment:
Right now it will error out saying "Invalid build type", which I think is probably fine since no one should be calling this with build, and a no-op wouldn't actually be bumping the build which could be confusing to someone expecting to see a file change as the result of running this command